stl/src/tzdb.cpp

```diff
@@ -55,7 +55,7 @@ namespace {
 
     [[nodiscard]] _Icu_api_level _Init_icu_functions(_Icu_api_level _Level) noexcept {
         while (!_Icu_functions._Api_level.compare_exchange_weak(
-            _Level, _Icu_api_level::_Detecting, _STD memory_order_acq_rel)) {
+            _Level, _Icu_api_level::_Detecting, _STD memory_order_acq_rel, _STD memory_order_acquire)) {

```

Comment:
[[atomics.types.operations]/23](https://eel.is/c++draft/atomics.types.operations#23) states "When only one `memory_order` argument is supplied, the value of [the first parameter] is `order`, and the value of [the second parameter] is `order` except that a value of `memory_order::acq_rel` shall be replaced by the value `memory_order::acquire` ..."

Consequently, this change is simply a longer way to spell what the existing code already does. I don't think the change is desirable.